helm_tests/airflow_aux/test_migrate_database_job.py

```diff
@@ -60,6 +60,18 @@ def test_should_support_annotations(self):
         assert "fiz" in job_annotations
         assert "fuz" == job_annotations["fiz"]
 
+    def test_should_add_component_specific_labels(self):
+        docs = render_chart(
+            values={
+                "migrateDatabaseJob": {
+                    "labels": {"test_label": "test_label_value"},
+                },
+            },
+            show_only=["templates/jobs/migrate-database-job.yaml"],
+        )
+        assert "test_label" in jmespath.search("spec.template.metadata.labels", docs[0])
+        assert jmespath.search("spec.template.metadata.labels", docs[0])["test_label"] == "test_label_value"
```

Comment:
could you add a second test to check if the labels and migrateDatabaseJob labels are merged correctly?

Reply:
I just added the new test, I hope it is what you had in mind?